server/apps/content_types/content_types.py

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
I would suggest renaming the file to resource.py to be in line with the convention we used most of the time.